packages/amazon-ssm-agent/amazon-ssm-agent.spec

```diff
@@ -35,11 +40,39 @@ Conflicts: (%{_cross_os}image-feature(no-fips) or %{name}-bin)
 %description fips-bin
 %{summary}.
 
+%package plugin
+Summary: A statically-linked agent to enable remote management of EC2 instances
+Requires: %{name}-plugin(binaries)
+
+%description plugin
+%{summary}.
+
+%package plugin-bin
+Summary: Statically-linked remote management agent binaries
+Provides: %{name}-plugin(binaries)
+Requires: (%{_cross_os}image-feature(no-fips) and %{name}-plugin)
+Conflicts: (%{_cross_os}image-feature(fips) or %{name}-plugin-fips-bin)
+
+%description plugin-bin
+%{summary}.
+
+%package plugin-fips-bin
+Summary: Statically-linked remote management agent binaries, FIPS edition
+Provides: %{name}-plugin(binaries)
+Requires: (%{_cross_os}image-feature(fips) and %{name}-plugin)
+Conflicts: (%{_cross_os}image-feature(no-fips) or %{name}-plugin-bin)
+
+%description plugin-fips-bin
+%{summary}.
+
 %prep
 %setup -n %{gorepo}-%{version}
+%global __scm git
```

Comment:
You should do what you did in socat:

https://github.com/bottlerocket-os/bottlerocket-core-kit/blob/8efc145a8cf7f6337ac5219cdb9b3da431be82e5/packages/socat/socat.spec#L16

Reply:
Ack